src/Symfony/Component/Messenger/Retry/MultiplierRetryStrategy.php

```diff
@@ -39,12 +39,12 @@ class MultiplierRetryStrategy implements RetryStrategyInterface
     private $maxDelayMilliseconds;
 
     /**
-     * @param int   $maxRetries           The maximum number of time to retry (null means indefinitely)
+     * @param int   $maxRetries           The maximum number of time to retry (-1 means indefinitely)
      * @param int   $delayMilliseconds    Amount of time to delay (or the initial value when multiplier is used)
      * @param float $multiplier           Multiplier to apply to the delay each time a retry occurs
      * @param int   $maxDelayMilliseconds Maximum delay to allow (0 means no maximum)
      */
-    public function __construct(?int $maxRetries = 3, int $delayMilliseconds = 1000, float $multiplier = 1, int $maxDelayMilliseconds = 0)
+    public function __construct(int $maxRetries = 3, int $delayMilliseconds = 1000, float $multiplier = 1, int $maxDelayMilliseconds = 0)
```

Comment:
this is a bc break that we cannot accept as a bug fix. please change it so that null can be passed to max_retries configuration.

Reply:
As this is in an experimental state it should be possible to do this change